cmd/keystore.go

```diff
@@ -160,6 +167,36 @@ func (k *keystoreState) importKey(a *appState) *cobra.Command {
 	return importCmd
 }
 
+// converts private key to keystore
+func (k *keystoreState) convertKey(a *appState) *cobra.Command {
+	convertCmd := &cobra.Command{
+		Use:   "convert",
+		Short: "convert keystore",
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Can you add example as well ? that would make the usage more clear.

Reply:
done